src/components/item-details/place-bet.tsx

```diff
@@ -26,7 +26,7 @@ const PlaceBet = ({ highestBid, actionDate, btnColor, className, item }: Props)
           Buy
         </Button>
       </div>
-      <PlaceBidModal show={showBidModal} handleModal={handleBidModal} item={item} />
+      <PlaceBidModal show={showBidModal} handleModal={handleBidModal} />
```

Comment:
Let's change the 'bid' name for something more meaningful (we are not bidding here)